githubutil/auth.go

```diff
@@ -129,10 +129,15 @@ func (a GitHubAppAuther) InsertHTTPAuth(req *http.Request) error {
 
 func (a GitHubAppAuther) GetIdentity() (string, error) {
 	ctx := context.Background()
-	client, err := NewInstallationClient(ctx, a.ClientID, a.InstallationID, a.PrivateKey)
```

Comment:
Make a new func called `NewAppClient` that explains the difference in its comment? Then `GenerateInstallationToken` can share it.

Reply:
done